Do not dynamically try to get the version from package metadata,
that fails the way we install the package.

Index: puppetboard/version.py
--- puppetboard/version.py.orig
+++ puppetboard/version.py
@@ -3,4 +3,4 @@
 #
 
 import importlib.metadata
-__version__ = importlib.metadata.version('puppetboard')
+__version__ = "${MODPY_DISTV}"
